src/border.jl

```diff
@@ -70,25 +142,68 @@ end
 borderinstance(b::AbstractBorder) = b
 
 """
+```julia
     Pad(style::Symbol, m, n, ...)
+```
+
+Construct an instance of [`Pad`](@ref) such that the image is prepended and appended symmetrically with `m` pixels at the lower and upper edge of dimension 1, `n` pixels for dimension 2, and so forth.
+
+#### Usage illustration
+Use `Pad(:replicate,2,4)` to designate that the top and bottom border should be
+replicated by two pixels, and the left and right border by four pixels.
+
+
+---
 
-Pad the input image symmetrically, `m` pixels at the lower and upper edge of dimension 1, `n` pixels for dimension 2, and so forth.
 """
 Pad(style::Symbol, both::Int...) = Pad(style, both, both)
 Pad(both::Int...) = Pad(:replicate, both, both)
 
 """
+```julia
     Pad(style::Symbol, (m,n))
+```
+Construct an instance of [`Pad`](@ref) such that the image is prepended and appended symmetrically with `m` pixels at the lower and upper edge of dimension 1, and `n` pixels for dimension 2.
```

Comment:
One option you can consider is to write the docstring like this:
```julia
"""
    Pad(style::Symbol, m, n, ...)
    Pad(style::Symbol, (m, n, ...))

Construct an instance of ...
"""
```
One of the things I like about this style is that is reduces the amount of text that users have to read, because the main explanation is stated only once.

But this is your choice, and I'll merge whatever you prefer.

Reply:
I followed your suggestion.